Ventivader/Ventivader/Utilities/HKBloodType+Readable.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  HKBloodType+Readable.swift
+//  Ventivader
+//
+//  Created by Al on 10/05/20.
+//  Copyright © 2020. All rights reserved.
+//
+import HealthKit
+
+extension HKBloodType {
+    static var allValues: [HKBloodType] {
+        return [.abNegative, .abPositive, .aNegative, .aPositive, .bNegative, .bPositive , .oNegative, .oPositive]
```

Comment:
very cool :)

Reply:
How do you plan on using the Blood Type? Just in the patient profile?

Reply:
Yes, it is only in the patient profile. 
Because maybe doctors care. Otherwise, I can remove it